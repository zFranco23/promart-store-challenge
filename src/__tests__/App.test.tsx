import { shallow } from 'enzyme';
import App from '../App';

import toJson from 'enzyme-to-json';

describe('Test in App.tsx', () => {
  test('App should render correctly', () => {
    const wrapper = shallow(<App />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
