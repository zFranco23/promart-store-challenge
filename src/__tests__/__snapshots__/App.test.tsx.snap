// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test in App.tsx App should render correctly 1`] = `
<Provider
  store={
    {
      "@@observable": [Function],
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
>
  <AppRouter />
</Provider>
`;
